resources/migrations/000_migrations.yaml

```diff
@@ -8513,6 +8513,8 @@ databaseChangeLog:
       validCheckSum:
         - 8:d60aa66f5909d37393ee711dea6996c7 # original checksum under #17649
         - 8:98f3ccbb3e8efc46f5766fcd51f39d0e # changed creator_id to nullable in #19398
+        - 8:ccfed82cd5d09891c7a380b428e5fbc5 # no idea where these two values come from :(
```

Comment:
@kdoh should we remove these before merging? Not sure what is this

Reply:
Ah yes that was a fix @jeff303 did that I cherry-picked because of a migration issue I was running into locally. Not sure if it hurts anything to include or not?

Reply:
Talked to Jeff and that change already went into main so I just dropped that commit and force pushed to update this branch.